### PR TITLE
[chore](profile) add task state in profile

### PR DESCRIPTION
**Pull request by mrhhsg**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by mrhhsg** [comment]
run buildall

File: be/src/pipeline/pipeline_task.cpp [hunk 1]
```diff
@@ -391,6 +399,7 @@ Status PipelineTask::execute(bool* eos) {
             *eos = status.is<ErrorCode::END_OF_FILE>() ? true : *eos;
             if (*eos) { // just return, the scheduler will do finish work
                 _eos = true;
+                _task_profile->add_info_string("TaskState", "EOS");
```
Comment (yiguolei) [inline be/src/pipeline/pipeline_task.cpp:402]: rename to FINISHED

File: be/src/pipeline/exec/scan_operator.cpp [hunk 2]
```diff
@@ -1284,6 +1284,7 @@ Status ScanOperatorX<LocalStateType>::get_block(RuntimeState* state, vectorized:
     if (*eos) {
         // reach limit, stop the scanners.
         local_state._scanner_ctx->stop_scanners(state);
+        local_state._scanner_profile->add_info_string("EOS", "true");
```
Comment (yiguolei) [inline be/src/pipeline/exec/scan_operator.cpp:1287]: True

**Comment by mrhhsg** [comment]
run buildall

**Comment by mrhhsg** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by jacktengg — APPROVED** [review]

**Pull request merged**